src/main/java/com/leggeart/legge/LawsuitServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.leggeart.legge;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.Optional;
+
+public class LawsuitServiceImpl implements  LawsuitService{
```

Comment:
IMO you don't have to add another class, you could have all those methods in LawsuitService.
Also, you need to add annotations at the top - @Service